contribute/index.md

```diff
@@ -101,3 +101,31 @@ various contributor activities:
 </div>
 </div>
 </div>
+
+### Tools and Libraries
+
+The Scala ecosystem includes a great many diverse open-source projects
+with their own maintainers and community of contributors.  Helping out
+one of these projects is another way to help Scala.  Consider lending
+on a hand on a project you're already using.  Or, to find out about
+other projects, see the
+[Libraries and Tools section](../community/index.html#community-libraries-and-tools)
+on our Community page.
+
+### Scala Community Build
+
+The Scala community build enables us to build and test a corpus of
```

Comment:
I would instead say:

> The Scala community build enables the Scala compiler team to build and test a corpus of Scala open source projects all together against development versions of the Scala compiler and standard library in order to discover regressions prior to releases. We use Typesafe's multi-project build tool, called [dbuild](https://github.com/typesafehub/dbuild)
+(which leverages [sbt](http://www.scala-sbt.org)), to build this corpus of Scala projects all together.

> If you're the maintainer ...